vendor/engines/split_accounts/app/models/split_accounts/split_account.rb

```diff
@@ -36,10 +39,30 @@ def extra_penny_count
     #   subaccounts.where("expires_at IS NOT NULL").order(expires_at: :asc).first
     #
     def earliest_expiring_subaccount
-      subaccounts = splits.map{ |split| split.subaccount }.select(&:expires_at)
+      subaccounts = splits.map(&:subaccount).select(&:expires_at)
       subaccounts.sort_by(&:expires_at).first
     end
 
+    def outmost_suspended_subaccount
+      subaccounts = splits.map(&:subaccount).select(&:suspended_at)
+      subaccounts.sort_by(&:suspended_at).last
```

Comment:
That makes sense. However, wouldn't that be the earliest `suspended_at`? Another option might be to `||=` suspended_at so it only gets updated if it wasn't already suspended.